pkg/middleware/scope_test.go

```diff
@@ -5,10 +5,13 @@ import (
 	"net/http/httptest"
 
 	middlewareapi "github.com/oauth2-proxy/oauth2-proxy/v7/pkg/apis/middleware"
+	requestutil "github.com/oauth2-proxy/oauth2-proxy/v7/pkg/requests/util"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+const requestID = "11111111-2222-4333-8444-555555555555"
```

Comment:
Same as before, can we rename?
```suggestion
const testRequestID = "11111111-2222-4333-8444-555555555555"
```